### ISSUE TITLE
types missing?

### ISSUE THREAD

**Issue by nicolasburtey** [vitalets/react-native-extended-stylesheet#133]
I'm losing types suggestion from Visual Studio Code if I use EStyleSheet. Do I need to install anything beyond "react-native-extended-stylesheet" to get the types as I have them with StyleSheet?

#### Environment
System:
    OS: macOS 10.15.3
    CPU: (8) x64 Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz
    Memory: 273.95 MB / 16.00 GB
    Shell: 5.0.16 - /usr/local/bin/bash
  Binaries:
    Node: 10.16.3 - ~/.nvm/versions/node/v10.16.3/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.13.1 - ~/.nvm/versions/node/v10.16.3/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 13.4, DriverKit 19.0, macOS 10.15, tvOS 13.4, watchOS 6.2
  IDEs:
    Android Studio: 3.5 AI-191.8026.42.35.5791312
    Xcode: 11.4.1/11E503a - /usr/bin/xcodebuild
  npmPackages:
    react: 16.13.1 => 16.13.1 
    react-native: 0.61.5 => 0.61.5

**Comment by vitalets** [comment]
> Do I need to install anything beyond "react-native-extended-stylesheet" to get the types as I have them with StyleSheet?

No. Types are included in [index.d.ts](https://github.com/vitalets/react-native-extended-stylesheet/blob/master/types/index.d.ts). Could you check with some other package having types in `index.d.ts`?

**Comment by neXorianus** [comment]
can confirm this in Visual Studio Code.
React.StyleSheet.create():  
![image](https://user-images.githubusercontent.com/1299616/81073419-e6dc2c80-8ee7-11ea-85df-624819467246.png)
EStyleSheet.create():
![image](https://user-images.githubusercontent.com/1299616/81073547-1428da80-8ee8-11ea-8ff6-cfee2e214bc7.png)

The problem is, that you define the properties of the create object as 
```ts
type AnyObject = {[key: string]: any};
```
(this plugin v0.12.0)
i guess, this should rather expand the React.StyleSheet types: 
```ts
type NamedStyles<T> = { [P in keyof T]: ViewStyle | TextStyle | ImageStyle };
```
(rn v0.60.31)

**Comment by neXorianus** [comment]
this is not solved in behalf of the opening post.

**Comment by neXorianus** [comment]
push

**Comment by erzzo** [comment]
I'm experiencing the same issue
